tests/testthat/test_read.any.R

```diff
@@ -133,22 +133,22 @@ test_that( 'times read in properly', {
 #  
 #})
 
-test_that( 'read xlsb', {
+# test_that( 'read xlsb', {
```

Comment:
please allow this test to run if readxlsb is installed.

Reply:
Has been updated to run if package is installed.